### ISSUE TITLE
Umbraco forms 8.73

### ISSUE THREAD

**Issue by lori698** [umbraco/Umbraco.Forms.Issues#567]
Just downloaded the new release went to copy a form I dont seem able to - cant click the copy button.
office/backoffice/UmbracoForms/Form/CopyForm:1 Failed to load resource: the server responded with a status of 500 ()

**Comment by AndyButland** [comment]
Hi - I'm not able to see this issue in testing having installed the latest version.  Could you check please you don't have any cached assets via a hard refresh after ["bumping the client dependency version"](https://adolfi.dev/blog/umbraco-bump-cdf/)?

Assuming you still see the problem are you able to share what the underlying 500 error is?

**Comment by AndyButland** [comment]
Thanks @OwainWilliams - that's annoying, and a little frustrating as unfortunately I still can't seem to replicate it.  Part of your stack trace even looks correct - i.e. `{"guid":"9d82c7ea-798f-4b9c-b9a5-150892d4d702","newName":"sf","copyWorkflows":false}` is passing a boolean value for the `copyWorkflows` option..

**Comment by AndyButland** [comment]
OK, think we've found it... I think you must have this value missing in `UmbracoForms.config`:

```
<setting key="StoreUmbracoFormsInDb" value="False" />
```

This was added in an earlier version to give the option of storing forms in the database or on disk, with the default if not present being to store on disk.  So if you add this setting into the file, with a value of false, that should behave as it did when the setting was missing, but will resolve this issue with the copy form function that is currently expecting that setting to be there.

We'll patch this for the next release such that it's not required to have in config any more, but meantime as a work around, if you just make sure this setting is present the copy form function should work again without error.
